sourced/ml/transformers/tfidf.py

```diff
@@ -0,0 +1,63 @@
+from collections import namedtuple
+import typing
+
+import pyspark
+
+from sourced.ml.algorithms import log_tf_log_idf
+from sourced.ml.transformers.transformer import Transformer
+
+TFIDF_Row = namedtuple("TDIDF_Row", ["token", "document", "value"])
+
+
+class TFIDF(Transformer):
+    """
+    Converts a single UAST into the weighted set (dictionary), where elements are strings
+    and the values are floats. The derived classes must implement uast_to_bag().
+    """
+    DEFAULT_DOCFREQ_THRESHOLD = 5
```

Comment:
Would be nice to remove it here and take from BagsExtractor in `__init__`. Or vice versa